protostar/testing/environments/fuzz_test_execution_environment.py

```diff
@@ -37,6 +37,22 @@
 )
 
 
+def multiply_decorator(other_dec, args):
```

Comment:
[Style] This function could be moved to a different file.

Reply:
[move multiply decorator to a separate file](https://github.com/software-mansion/protostar/pull/947/commits/7443eba804bd76a54ffd481ae41bb185d5872ffa)